inlong-agent/conf/agent.properties

```diff
@@ -99,4 +100,4 @@ agent.prometheus.exporter.port=9080
 # whether to enable audit
 audit.enable=true
 # audit proxy address
-audit.proxys=127.0.0.1:10081
\ No newline at end of file
+audit.proxys.null=
```

Comment:
variable name "audit.proxys.null" may be confusing

Reply:
fix it next issue